### PR TITLE
update hover list events

### LINKED ISSUE #67: CAMBIAR LA REACCIÓN DEL COLOR DEL TEXTO / CHANGE THE COLOR REACTION OF THE TEXT.
##Explanation.
En la lista de eventos encontramos los títulos,  al pasar el cursor  por ellos cambia de color negro a color azul, debería tener una reacción diferente que no sea  el color del texto  ejemplo . solo el titulo del evento me parece correcto un subrayado

In the list of events we find the titles, when passing the cursor over them it changes from black to blue color, it should have a different reaction other than the color of the text example . only the title of the event seems to me correct an underline

###Checklist

En este link  puede entrar y ver.

https://www.did-admin.tk/eventos

Historia de usuario /User istory

Como usuario quiero que la lista de eventos al pasar el cursor tenga una reacción diferente que no sea el cambio de color del  texto.

As a user I want the event list to have a different reaction when hovering the cursor other than changing the color of the text.

criterios de aceptación.

 lao textos de los eventos tendrán una reacción diferente al de cambiar de color

 event texts will have a different reaction when changing color.

Casos de uso.

# Referencias

- Por favor terminar en 3 días. Mas información: https://github.com/Azordev/did-admin-panel/issues/192

### PR DESCRIPTION
**Pull request by luleheco22**
### Description

Please include a summary of the change.

fixes # (#67)

#### Screenshots
image

If applicable, add screenshots about your changes

#### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How to test it

If applicable, describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

### Checklist:

- [ ] Doesn't break the current code.
- [ ] Passes linters and test, also is building.
- [ ] Doesn't have spelling or grammatical problems.
- [ ] Doesn't have unnecessary comments or debugging code.
- [ ] The branch is updated with the last dev branch changes.

### CODE REVIEW

**Review by Luisparr14 — COMMENTED** [review]
The function to delete the event doest work yet

**Comment by e-azocar** [comment]
@angelik0828 is this the expected result?
image

File: styles/Events.module.scss [hunk 1]
```diff
@@ -44,6 +44,15 @@
   grid-template-rows: 1fr 1fr 1rem;
   grid-template-columns: 200px 1fr;
   text-decoration: none;
+  transition: .3s ease;
```
Comment (heliomar-pena) [inline styles/Events.module.scss:47]: Please add the elements that expect to be changed https://developer.mozilla.org/en-US/docs/Web/CSS/transition-property

File: styles/Events.module.scss [hunk 2]
```diff
@@ -44,6 +44,15 @@
...
+  &:hover{
+    color: $white;
+    background: $black;
+    box-shadow: inset 0 0 20px;
+    outline-color: rgba(49, 138, 172, 0);
+    outline-offset: 80px;
+    text-shadow: 1px 1px 6px $white;
```
Comment (heliomar-pena) [inline styles/Events.module.scss]: I think that text-shadow is not needed, @angelik0828 please confirm, here's some examples about what text-shadow is: https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow

imagen
imagen

This makes the text looks like is shining

File: styles/Events.module.scss [hunk 3]
```diff
@@ -44,6 +44,15 @@
   grid-template-rows: 1fr 1fr 1rem;
   grid-template-columns: 200px 1fr;
   text-decoration: none;
+  transition: .3s ease;
+  &:hover{
+    color: $white;
+    background: $black;
+    box-shadow: inset 0 0 20px;
```
Comment (heliomar-pena) [inline styles/Events.module.scss]: If you don't add any color, box-shadow will inherit the "color" property's value, what is white currently, in this case the background is white, so the box-shadow is not visible, is the box shadow really needed?, in that case, what color should be?

here's an example of how box-shadow works:

lastrecord

CC: @angelik0828

File: styles/Events.module.scss [hunk 4]
```diff
@@ -44,6 +44,15 @@
...
+    outline-color: rgba(49, 138, 172, 0);
+    outline-offset: 80px;
```
Comment (heliomar-pena) [inline styles/Events.module.scss]: Due to you are using 0 in the alpha channel, outline-color is creating an "invisible" outline, is actually needed to have an outline so? I meant, if it's not visible, maybe can be removed these lines

lastrecord

**Review by Eghinner — CHANGES_REQUESTED** [review]
Issues only recommended a hover effect to the title

**Comment by Eghinner** [comment]
hmmm, take this reference https://laravel-news.com

**Comment by angelik0828** [comment]
## ⚠️  According to what task 67 mentions, the idea was to make the change in the side menu, place the states in the buttons. 

image

## ✔️ But since this change has already been made and is currently needed in my browser, it looks like this:


https://user-images.githubusercontent.com/4030477/188912247-cb15b710-5d87-4bdd-a1a6-9fb1c897b340.mp4


I think it looks good

**Review by angelik0828 — APPROVED** [review]

**Comment by Luisparr14** [comment]
@luleheco22 the delete button does not work (issue #118), please resolve or remove from description.

**Comment by angelik0828** [comment]
## ❌ Delete button and validation modal not working

image

**Comment by Eghinner** [comment]
@angelik0828 @luleheco22 hover dahsboard hover is resolved in another issue https://github.com/Azordev/did-admin-panel/issues/183

**Comment by angelik0828** [comment]
> @angelik0828 @luleheco22hover dashsboard hover se resuelve en otro problema #183

understood we can continue!!!

**Review by angelik0828 — APPROVED** [review]

**Comment by luleheco22** [comment]
I can't find a way to do the hover with the event card, it only takes them when the hover is done in the title, if you can suggest what change I can make in the code, I'll stay tuned

File: styles/Events.module.scss [hunk 5]
```diff
@@ -46,32 +46,32 @@
...
-    }
-    &:hover:after{
-        transform: scaleX(1);
-        transform-origin: bottom left;
-    }
+}
+
+.event-item:hover{
+   color: $black;
+  .line{
```
Comment (Eghinner) [inline styles/Events.module.scss]: move this to father simply. without hove, I mean

File: styles/Events.module.scss [hunk 6]
```diff
@@ -46,32 +46,32 @@
...
+    display: inline-block;
+    position: relative;
+  }
+  .line:after {
```
Comment (Eghinner) [inline styles/Events.module.scss]: move this to father

File: styles/Events.module.scss [hunk 7]
```diff
@@ -46,32 +46,32 @@
...
+    width: 100%;
+    transform: scaleX(0);
+    height: 4px;
+    bottom: 0;
+    left: 0;
+    background-color: $black;
+    transform-origin: bottom right;
+    transition: transform 0.6s ease-out;
+  }
+  .line:hover:after {
```
Comment (Eghinner) [inline styles/Events.module.scss]: remove hover

**Review by Eghinner — APPROVED** [review]

**Pull request merged**